OpenOversight/app/utils.py

```diff
@@ -95,6 +96,18 @@ def allowed_file(filename):
            filename.rsplit('.', 1)[1].lower() in current_app.config['ALLOWED_EXTENSIONS']
 
 
+def requires_conversion(filename):
+    return '.' in filename and \
+        filename.split('.', 1)[1].lower() in current_app.config['REQUIRES_CONV']
+
+
+def convert_image(file_to_upload):
+    changeimage = Image.open(file_to_upload.filename)
```

Comment:
Since we're calling Image.open() on untrusted content here, can you also hoist the decompression-bomb-safety warning into an error as documented in [PIL's docs](https://pillow.readthedocs.io/en/3.1.x/reference/Image.html#PIL.Image.open)? AIUI we've been mostly free of risk-to-the-server for the files we're uploading since we were only reading headers out of them before, but now we're doing parsing and things that risk actual attack.